postgres-matrix.yml

```diff
@@ -3,14 +3,14 @@ project_name: citus
 
 version_matrix:
   - 8.0:
-      postgres_versions: [10, 11]
+      postgres_versions: [ 10, 11 ]
   - 9.0:
-      postgres_versions: [11, 12]
+      postgres_versions: [ 11, 12 ]
   - 9.5:
-      postgres_versions: [11, 12, 13]
+      postgres_versions: [ 11, 12, 13 ]
   - 10.1:
-      postgres_versions: [12, 13]
+      postgres_versions: [ 12, 13 ]
   - 10.2:
-      postgres_versions: [12, 13, 14]
+      postgres_versions: [ 12, 13, 14 ]
   - 11.0:
-      postgres_versions: [13,14]
+      postgres_versions: [ 13, 14, 15 ]
```

Comment:
I believe 11.1 will be the first version to support PG15.

Reply:
I added this for nightlies. After 11.1 release I will change it. This is for beta process

Reply:
We use `11.1devel` version on the main branch.

1. Why do you need to update entries for `11.0` for nightlies? Should not that be 11.1?
2. Would not this change cause problems if we want to release a patch version for 11.0 that does not and will not support PG15.

I suggest we add 11.1 here and make sure nightlies actually use the values for that one.

Reply:
Current structure does not work like that. Nightlies are being processed seperately and evaluated with the current branch. Actually we do not make a version check since it is on main branch.